mfs/ops.go

```diff
@@ -172,12 +171,11 @@ func Mkdir(r *Root, pth string, opts MkdirOpts) error {
 	return nil
 }
 
+// Lookup extracts the root directory and performs a lookup under it.
+// TODO: Now that the root is always a directory, can this function
+// be collapsed with `DirLookup`? Or at least be made a method of `Root`?
```

Comment:
This function is used in many places within the code.  I would be okay with making this a method of Root but would not collapse it with `DirLookup` and `DirLookup` will work on any directory and not just the root.